Library/Formula/foma.rb

```diff
@@ -0,0 +1,15 @@
+class Foma < Formula
+  desc "Finite-state compiler and C library"
+  homepage "https://code.google.com/p/foma/"
+  url "https://bitbucket.org/mhulden/foma/downloads/foma-0.9.18.tar.gz"
+  sha256 "cb380f43e86fc7b3d4e43186db3e7cff8f2417e18ea69cc991e466a3907d8cbd"
+
+  def install
+    system "make"
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/foma", "-v"
```

Comment:
Could this test be modified to do something more substantial than e.g. `-v`/`-h`? Thanks!